ofcourse-templates/master.html

```diff
@@ -1,8 +1,14 @@
 <!DOCTYPE html>
+{% macro title() %}
+  {{course.name}}@{{course.place}}
+{% endmacro %}
+{% macro glyphicon(glyph) %}
+<span class="glyphicon glyphicon-{{glyph}}"></span>
```

Comment:
I don't really see a need to move it to the Python code, since it's VERY template-dependent. The other option is to move it (and maybe other macros) into a separate file, but it seems like little benefit